pkg/api/client/informers/externalversions/generic.go

```diff
@@ -62,6 +63,10 @@ func (f *sharedInformerFactory) ForResource(resource schema.GroupVersionResource
 	case v1alpha1.SchemeGroupVersion.WithResource("subscriptions"):
 		return &genericInformer{resource: resource.GroupResource(), informer: f.Operators().V1alpha1().Subscriptions().Informer()}, nil
 
+		// Group=operators.coreos.com, Version=v1alpha2
```

Comment:
did you add this manually? it should have been generated

Reply:
No, the v1alpha1 generation produced a comment like this too.